### PR TITLE
Set GC params for Java Stress Test

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
For https://github.com/grpc/grpc-java/issues/1812

### CODE REVIEW

File: tools/run_tests/stress_test/configs/java.json [hunk 1]
```diff
@@ -24,6 +24,9 @@
         }
       }
     },
+    "env": {
```
Comment (sreecha) [inline tools/run_tests/stress_test/configs/java.json]: This "env" should be inside the scope of a template (in this case inside the scope of "default"). 
Move this next to the "metricsArgs" above.

Comment (carl-mastrangelo) [inline tools/run_tests/stress_test/configs/java.json]: Are you sure about the scoping?  In the python version "env" is a peer of baseTemplates, as it is here.

Comment (carl-mastrangelo) [inline tools/run_tests/stress_test/configs/java.json]: Also done for the server args.

Comment (sreecha) [inline tools/run_tests/stress_test/configs/java.json]: Yes, I am sure... 
https://github.com/grpc/grpc/blob/master/tools/run_tests/stress_test/configs/python.json#L25
https://github.com/grpc/grpc/blob/master/tools/run_tests/stress_test/configs/python.json#L51

**Comment by sreecha** [comment]
LGTM. thanks.

**Comment by carl-mastrangelo** [comment]
@jtattermusch or @nicolasnoble  

Could I get a merge?

**Comment by jtattermusch** [comment]
I need all tests green except known flakes.

**Comment by carl-mastrangelo** [comment]
Please check.

**Comment by sreecha** [comment]
Can be merged now.. known flakes.. 
(Also, none of those tests are really related to this change)

**Pull request merged**